packages/auth-remix/src/server.ts

```diff
@@ -1169,6 +1188,27 @@ function _extractParams(
   return params;
 }
 
+function parseCookies(
+  req: Request,
+  options: {
+    baseUrl: string;
+    authRoutesPath: string;
+    authCookieName: string;
+    pkceVerifierCookieName: string;
+    passwordResetPath?: string;
+    magicLinkFailurePath?: string;
+  },
```

Comment:
You don't need to define all of the options here, just the ones you use.

```suggestion
  options: {
    authCookieName: string;
    pkceVerifierCookieName: string;
  },
```

---

Another option is to make this a method on the class so you can use `this.options` instead of passing it in, but maybe this would need to be defined on multiple classes?